tests/output/test_bazefetcher.py

```diff
@@ -1,69 +1,198 @@
 from datetime import datetime
+from datetime import timedelta
 from pytz import utc
-import camille.output
-import camille.source
+from pytz import timezone
+from camille.output import bazefetcher as bazeoutput
+from camille.source import bazefetcher as bazesource
 import numpy as np
 import os
 import pandas as pd
 
+_test = 'test'
```

Comment:
After our discussion in general decided to go with something similar to the following: 
Use variable where 
1) future change is likely
2) or variable value is not simple (long number of string)
3) or value is used so many times that I got tired of typing it :)
Otherwise will use explicit value in order to improve readability
Changed